composer.json

```diff
@@ -1,30 +1,27 @@
 {
-    "name": "shopgate/connect-integration-sdk",
+    "name": "shopgate/connect-integration-sdk-php",
     "description": "A PHP framework supporting the integration with the Shogate Connect Platform.",
     "homepage": "https://github.com/shopgate/connect-integration-sdk",
     "license": "Apache-2.0",
     "version": "1.0.0-alpha",
-    "config": {
-        "platform": {
-            "php": "5.3.3"
-        }
-    },
     "authors": [
         {
             "name": "Shopgate",
             "email": "interfaces@shopgate.com"
         }
     ],
     "require": {
-        "php": ">=5.3",
+        "php": ">=5.4",
         "zendframework/zend-json": "2.2.*"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.8.*"
+        "phpunit/phpunit": ">=4.8 <8",
```

Comment:
should not do this eiether, these versions are mutually exclusive, as I mentioned, we cannot have this bound so low, 4.8 uses PSR-0, while 6+ uses PSR-4. Just use the latest and greatest with whichever you are testing right now. Are you testing with 8? Plop that version in as `^8.0.0` if 7 something, then use that.

Reply:
I need version 7, because version 8 is not compatible with php 5.6

Reply:
just defined `^7.0`, it will prevent 8.0>

Reply:
I'll go back to <8
I don't see the advantage of "phpunit/phpunit": "^5.0.0 ||^6.0.0 || ^7.0.0"